transcripts/4.3.2 Selecting Quality Packages Part 2.txt

```diff
@@ -12,39 +12,39 @@
-00:53 Alright, I am going to try and find the readme file for the Reqests library now. 
+00:49 Let's take a look at a real project README now. 
+00:53 Alright, I am going to try and find the README file for the Reqests library now. 
 00:57 So, typically, what you'd be looking for is a link to the project source repository 
-01:02 so it already looks like this is hosted on github here, 
+01:02 so it already looks like this is hosted on GitHub here, 
 01:05 so I am just going to look for a link.
-01:09 Alright, there we go, requests at github, 
-01:12 so that should be the link to the github project 
-01:14 where we can check out the project readme, yes, this is it, 
-01:16 so when I scroll down this is where github displays the readme file, 
-01:19 and for other source control websites like bitbucket, 
+01:09 Alright, there we go, requests at GitHub, 
+01:12 so that should be the link to the GitHub project 
+01:14 where we can check out the project README, yes, this is it, 
+01:16 so when I scroll down this is where GitHub displays the README file, 
+01:19 and for other source control websites like Bitbucket, 
 01:22 they will either display the readme in the same fashion 
-01:25 or you can view it by finding the readme file and then clicking directly on that. 
+01:25 or you can view it by finding the README file and then clicking directly on that. 
 01:29 Now the first thing that I can see here is that this looks really similar 
 01:32 to the project homepage which isn't a bad thing, 
-01:35 I mean, this contains all of the information that I wanted to get out of the readme 
+01:35 I mean, this contains all of the information that I wanted to get out of the README 
 01:38 and it looks like it's really well structured and nicely formatted. 
 01:41 So this is great, this tells me how to install the library, and it looks really simple, 
 01:46 it's pointing me to the documentation 
 01:48 and it also tells me how to contribute to the project. 
-01:52 If you're wondering what should go into a great readme file, 
+01:52 If you're wondering what should go into a great README file, 
 01:54 I wrote this article a while ago,
-01:56 about how you can write a great readme for a github project, 
+01:56 about how you can write a great README for a github project, 
 01:59 I am covering a number of things here that in my opinion 
-02:01 should go into a great readme, for example, it should talk about 
+02:01 should go into a great README, for example, it should talk about 
 02:05 what the project actually does, how to install it, 
 02:07 some example usage, how someone could set up a development environment,
 02:11 some link to a change log, and then also license and author info, 
 02:16 you can check out the full article in the link that you see here. 
-02:24 Now let's go back to the requests readme file. I said that I'd like to know 
+02:24 Now let's go back to the Fequests README. I said that I'd like to know 
```

Comment:
Hi @TeddyBear06 The changes to this file look great. However, this one line seems to have gotten worse. :) I'm sure you mean Requests README but it's Fequests. Could you update this on your side and push it again to your repo to update the PR?